contact/lib/contact/model/autocomplete_capability.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:jmap_dart_client/http/converter/unsigned_int_nullable_converter.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability_properties.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'autocomplete_capability.g.dart';
+
+@JsonSerializable(
+  explicitToJson: true,
+  includeIfNull: false,
+  converters: [
+    UnsignedIntNullableConverter()
+  ]
+)
+class AutocompleteCapability extends CapabilityProperties {
```

Comment:
Should this be defined in `jmap-dart-client` instead?

Reply:
No, it is an extension so we have already created a module to handle it.